tests/foreman/ui/test_contenthost.py

```diff
@@ -1458,9 +1458,7 @@ def test_search_for_virt_who_hypervisors(session, default_location):
 @pytest.mark.destructive
 @pytest.mark.run_in_one_thread
 @pytest.mark.upgrade
-def test_content_access_after_stopped_foreman(
-    foreman_service_teardown, rhel7_contenthost, target_sat
-):
+def test_content_access_after_stopped_foreman(default_sat, rhel7_contenthost):
```

Comment:
I think your rebase didn't quite go in the right direction. `target_sat` is the correct fixture name now.